### PR TITLE
Add Teleport Team docs

### PR DESCRIPTION
**Pull request by ptgott**

### CODE REVIEW

**Review by xinding33 — APPROVED** [review]

File: docs/pages/choose-an-edition/teleport-team.mdx [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+```mermaid
+flowchart LR
+subgraph team["Teleport Team tenant"]
+  direction LR
+  proxy["Teleport Proxy Service"]
+  auth["Teleport Auth Service"]
+  proxy<-.->auth
+end
+
+subgraph public["The internet"]
```
Comment (alexfornuto) [inline docs/pages/choose-an-edition/teleport-team.mdx:28]: Consider giving this an empty title and no background. "The internet" might be best diagramed as open.

File: docs/pages/choose-an-edition/teleport-team.mdx [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+<Notice type="tip">
+
+For a detailed comparison of Teleport's editions, including how Teleport Team
+compares to Teleport Team, see the [Choose an Edition](introduction.mdx) guide.
+
+</Notice>
+
+## Get started with Teleport Team
+
+This section walks you through the process of registering a local server with a
```
Comment (alexfornuto) [inline docs/pages/choose-an-edition/teleport-team.mdx:50]: Unsure about "section" here, since this describes the bulk of the rest of the page.

File: docs/pages/choose-an-edition/teleport-team.mdx [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+  </Notice>
+
+- The `tsh` client tool. Visit our [Installation Guide](../installation.mdx)
+  for instructions. If you want to get started quickly, you can skip this for
+  now and access Teleport through your browser.
+
+### Step 1/4. Spin up your server
+
+Spin up a server on your workstation so you can register this server with your
+Teleport Team tenant:
```
Comment (alexfornuto) [inline docs/pages/choose-an-edition/teleport-team.mdx:82]: "server" is used redundantly. Consider something like:

```suggestion
Start a docker container on your workstation to register as a server in your
Teleport Team tenant:
```

File: docs/pages/choose-an-edition/teleport-team.mdx [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+
+```code
+$ docker run -it ubuntu:22.10 /bin/bash
+```
+
+Your terminal will begin a shell session in the `ubuntu:22.10` container. From
+there, make sure you have `curl` and `telnet` installed. When we run a script to install Teleport later in this guide, the script will require both of these tools.
```
Comment (alexfornuto) [inline docs/pages/choose-an-edition/teleport-team.mdx:89]: Line break and `s/./:`.

```suggestion
Your terminal will begin a shell session in the `ubuntu:22.10` container. From
there, make sure you have `curl` and `telnet` installed. When we run a script to
install Teleport later in this guide, the script will require both of these tools:
```

File: docs/pages/choose-an-edition/teleport-team.mdx [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+
+```code
+$ apt update && apt install -y curl telnet
+```
+
+The rest of the guide assumes that you will run it as a non-`root` user, but the
+shell in the container you created earlier is running as root. Alias `sudo` to
+override this:
```
Comment (alexfornuto) [inline docs/pages/choose-an-edition/teleport-team.mdx:97]: This sounds odd to me. If it's "the guide" which we wrote, then we could write it to not use sudo. Instead, can we rephrase this to say that the Teleport web UI provides example commands assuming a non-`root` user?

File: docs/pages/choose-an-edition/teleport-team.mdx [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+
+<Notice type="tip">
+
+If you get an error connecting to your container, interrupt the `teleport`
+process in your container's shell with **Ctrl-C**, then run `teleport start` to
+restart the process.
+
+</Notice>
+
+You should see a terminal prompt in your browser. Your Teleport Team
```
Comment (alexfornuto) [inline docs/pages/choose-an-edition/teleport-team.mdx:158]: Suggest specifying that this will happen in a new tab/window.

File: docs/pages/choose-an-edition/teleport-team.mdx [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+connects to your container via a reverse tunnel. 
+
+Reverse tunnels like this allow Teleport to manage access to resources like
+Kubernetes clusters and remote desktops, even if these resources are running on
+private networks.
+
+Execute some commands in your terminal. When you have finished, run `exit` to
+end your session.
+
+In the **Manage Access** view, click **Finish**, then **Browse existing
```
Comment (alexfornuto) [inline docs/pages/choose-an-edition/teleport-team.mdx:169]: Related to previous comment, suggest referencing the previous tab/window.

File: docs/pages/includes/edition-comparison.mdx [hunk 8]
```diff
@@ -1,65 +1,65 @@
...
-|Version support|All supported releases available to install and download.|All supported releases available to install and download.|Deploys last stable release with 2-3 week lag for stability.|
-|[Backend support](../reference/backends.mdx)|Any S3-compatible storage for session records, many managed backends for custom audit log storage.|Any S3-compatible storage for session records, many managed backends for custom audit log storage|All data is stored in DynamoDB and S3 with server-side encryption|
-|Data storage location|Can store data anywhere in the world, on most managed cloud backends|Can store data anywhere in the world, on most managed cloud backends| Data is stored in `us-west-2`, with Proxy Service instances deployed across the world for low-latency access|
-|[Hardware Security Module support](../choose-an-edition/teleport-enterprise/hsm.mdx) for encryption at rest|&#10006;|&#10004;|&#10006;|
+||Open Source|Enterprise|Cloud|Team|
+|---|---|---|---|---|
+|Auth and Proxy Service Management|Self-hosted|Self-hosted|Fully managed|Fully managed|
+|Proxy Service domain name|Custom|Custom|A subdomain of `teleport.sh`|Subdomain of `teleport.sh`|
```
Comment (alexfornuto) [inline docs/pages/includes/edition-comparison.mdx:47]: Suggest either adding "A" or removing it from the previous column for parity.

File: docs/pages/choose-an-edition/introduction.mdx [hunk 9]
```diff
@@ -8,14 +8,14 @@ which edition is most appropriate for your use case.
...
 
-### Teleport Community Edition
+### Teleport Team
 
-We provide a free, open source distribution of Teleport that enables you to get
-secure access to databases, Windows desktops, Kubernetes clusters, and more.
+Teleport Team makes it easier for small organizations to get started with
+enabling secure access to their infrastructure. It includes a subset of Teleport
+Enterprise Cloud features, and teams can switch to Teleport Enterprise Cloud
+as they scale up their Teleport usage.
```
[the remark this answers is not in the document]
Comment (ptgott) [reply docs/pages/choose-an-edition/introduction.mdx:16]: Ah, nice. I'll change to "scale up" in general, since it can imply adding users

File: docs/pages/choose-an-edition/teleport-team.mdx (hunk 5 above)
Comment (ptgott) [reply docs/pages/choose-an-edition/teleport-team.mdx:97]: To make this clearer/simpler, I've moved this text just before the instructions to copy the installation script:

````markdown
The Teleport installation script uses `sudo`, which is not installed on your
Docker container. Alias `sudo` in your container terminal to override this:

```code
$ alias sudo=""
```

Copy the script and paste it into the shell you opened earlier on your Docker
container.
````

**Review by alexfornuto — APPROVED** [review]
